ovs/lib/migration.py

```diff
@@ -664,3 +669,90 @@ def _update_manifest_cache_size(_proxy_config_key):
                 Configuration.register_usage(System.get_component_identifier(), registration_key)
 
         MigrationController._logger.info('Finished out of band migrations')
+
+        ####################################################
+        # deduplicate configs of proxies: now one config per remote alba-backend
```

Comment:
since when do we write our comments without a capital letter?
https://i.kym-cdn.com/photos/images/facebook/001/209/105/7b2.jpeg